.travis.yml

```diff
@@ -1,6 +1,7 @@
-language: nix
+language: minimal
```

Comment:
Would this work without `sudo`?

Reply:
As far as I know, `language: nix` uses a VM.

Reply:
> Would this work without `sudo`?

TIL: https://config.travis-ci.com/ref/sudo - `sudo` is deprecated and has no effect.